app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.kt

```diff
@@ -17,8 +23,12 @@ class LoginFragment private constructor() : WolmoFragment<FragmentLoginBinding,
 
             editTextEmail.addTextChangedListener {}
             editTextPassword.addTextChangedListener {}
+
             buttonLogin.setOnClickListener {
-                presenter.onClickLogin(editTextEmail.text.toString(), editTextPassword.text.toString())
+                val etEmail = editTextEmail
+                val etPassword = editTextPassword
+                val context = requireContext()
+                presenter.onClickLogin(editTextEmail.text.toString(), editTextPassword.text.toString(), etEmail, etPassword, context)
```

Comment:
You just need to pass the EditText content, not the entire component. Also, you shouldn't be using the Context from the presenter.